runtime/vm/jvminit.c

```diff
@@ -1868,7 +1868,35 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 				}
 			}
 #endif /* !defined(WIN32) && !defined(J9ZTPF) */
+			/* Parse options related to Large Pages */
+			{
+				IDATA argIndexUseLargePages = FIND_AND_CONSUME_ARG(EXACT_MATCH, MAPOPT_XXUSELARGEPAGES, NULL);
+				IDATA argIndexLargePageSizeInBytes = FIND_AND_CONSUME_ARG(STARTSWITH_MATCH, MAPOPT_XXLARGEPAGESIZEINBYTES_EQUALS, NULL);
+				
+				if (argIndexUseLargePages > argIndexLargePageSizeInBytes) {
+					vm->largePageArgIndex = argIndexUseLargePages;
+					vm->largePageSizeRequested = -1;
+				}
+				else if (argIndexLargePageSizeInBytes >= 0) {
```

Comment:
`(-1 != argIndexLargePageSizeInBytes)` would be cleaner check

Reply:
Makes sense. Thanks!

Reply:
See https://github.com/eclipse/openj9/pull/7476/commits/cad6beaec1cc182b1a5325f005ef62dba56d2810